Secure Score/Diagnostic Logs/Log Analytics/README.md

```diff
@@ -0,0 +1,32 @@
+# Enable diagnostic logging automatically on your resources
+
+This collection of samples are providing a better security logs to your resources by enabling Diagnostic Logs and retain them up to a year in Azure Policy. When configuring your diagnostic logs settings, you can export the logs to three targets.
+You can use each target that is not in the same subscription as the one emitting logs. The user who configures the setting must have the appropriate RBAC access to both subscriptions.
+
+## Storage Account
+
+You can export data to blob storage, export to CSV, and generate graphs in Excel.
+
+## Event Hub
+
+You can export data to Event Hubs and correlate with data from other Azure services.
+
+## Log Analytics
+
+You can export data to Azure Monitor logs and view data in your own Log Analytics workspace.
+
+[Learn more about policy definition structure](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure)
+
+# Contributing
```

Comment:
Should this be part of every readme file?

Reply:
The structure of diagnostic logs will include one main/general readme file describing the essence of diagnostic logs and specific file for each resource with unique link button to policy definition. 

> Should this be part of every readme file?
This link will only be in the main/general reame file